pkg/pod/resource_request.go

```diff
@@ -50,16 +52,42 @@ func resolveResourceRequests(containers []corev1.Container) []corev1.Container {
 		}
 	}
 
+	// Get limitrange minimum for container requests so they won't
+	// be zeroed out if minimum is specified in namespace
+	limitRangeItems := limitRange.Spec.Limits
+	min := allZeroQty()
+	if limitRange != emptyLimitRange {
```

Comment:
won't this compare the pointer addresses rather than the struct values?  I suspect that these addresses will always be different since we're allocing `emptyLimitRange` and `limitRange` in different places.  If that's not the case I'm a bit confused on what's happening.

Reply:
Yes, you're correct and thanks for pointing this out. I misinterpreted the limitrange here. Let me push up a change to have the default case be that limitrange is nil.